text/0431-guides-restructure.md

```diff
@@ -0,0 +1,414 @@
+
+Template Syntax will cover Ember Handlebars basics and built-in features like `{{#each}}` and `{{#if}}`. It will take care to make it clear that Ember uses its own implementation of Handlebars. It will clearly differentiate between control structures (loops and conditionals), inline vs block styles, component invocation, and helpers.
+
+Template helpers are split out and moved to "Leveling Up" because developers do not need to use them to write basic Ember apps (they are not a prerequisite). This section includes writing custom Helpers and using built-in helpers like `get`, `let`, and `array`. Although helpers themselves are not considered a difficult topic, they are used most often when developers understand their app's architecture and the flow of data. Also, template helpers have more in common with JavaScript functions than concerns relating to layout and data flow.
+
+Lastly, another reason to split out Template syntax from Template Helpers is because the syntax section is expected to expand with the addition of Octane content.
+
+An alternative to this split is to reduce the documentation of helpers themselves in the Guides, and lean on the API docs instead, however great care should be taken in removing documentation. Without a strong case for it, we lean towards leaving them in the Guides.
+
+#### Working with JavaScript
+
+A common concern of new learners is, what is JavaScript and what is special to Ember? The Working with JavaScript section incorporates the existing JavaScript Primer and new content that will help make the distinctions clearer. Ember.js has procatively adopted JavaScript APIs as they become available, such as Classes and Decorators, in some cases before the larger JavaScript community becomes familiar with them. Key people in the Ember Community are also involved in the development of JavaScript itself through TC39. To an extent, it is our responsibility to at least suggest to developers which JavaScript concepts they must learn in order to feel comfortable with Ember.
+
+#### Components
+
+Components will need the most refactor work and new content for Octane, so here’s the subtopic breakdown:
+
+- Creating a component (includes Component Types post-Octane)
+- Displaying data in a Template
+- Adding actions
+- Nesting components (includes suggestions of when to break things into components)
+- Passing arguments
+- Using computed properties (extremely basic example, link to the Computed Property section)
+- Working with arrays of data
+- Lifecycle hooks
+
+#### State management
+
+The State Management section is important because whether "computed properties" are available in their current state or as `@tracked` Decorators, the nunances must be within easy reach for existing Ember developers and future learners. We rename this topic in alignment with the Topic Naming convention mentioned above.
```

Comment:
```suggestion
The State Management section is important because whether "computed properties" are available in their current state or as `@tracked` Decorators, the nuances must be within easy reach for existing Ember developers and future learners. We rename this topic in alignment with the Topic Naming convention mentioned above.
```